src/send.js

```diff
@@ -29,7 +29,7 @@ export default async function send(core) {
  */
 async function post(config) {
   switch (true) {
-    case !!config.inputs.token:
+    case !!config.inputs.method:
```

Comment:
IMO an important change that might've been missed without testing a bit! Before changes, an environment variable might've set this and caused this `case` to run despite possible inputs preferring the `webhook`.

This is covered and confirmed in the `send.spec.js` tests!